examples/ml/gpt-2/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
+FROM tensorflow/tensorflow:1.14.0-gpu-py3
 
 RUN apt-get update && \
     apt-get install -y python3-pip && \
     rm -rf /var/lib/apt/lists/*
 
-RUN pip3 install twitterscraper gpt_2_simple tensorflow-gpu==1.14
+RUN pip3 install twitterscraper gpt_2_simple
```

Comment:
This is slightly different from what you had in master. I think this is what I wound up testing with and getting to work though so I figured I'd go with what was tested. Happy to change it if you think the other one is going to be more stable.

Reply:
lgtm, the issue originally was it was trying to use latest (which at the time was 1.15, which isn't compatible with cuda 10 (needs 10.1 i think?)
tldr; if this works and uses the GPU, let's go with it